content/html/encyclopedia/tables.md

```diff
@@ -0,0 +1,108 @@
+---
+Subjects:
+  - "HTML/CSS"
+  - "Web Development"
+Tags: 
+  - "Elements"
+  - "HTML & CSS"
+  - "Table"
+  - "Web Development"
+Catalog Content:  # If listing multiple URLs, please put the most relevant one first 
+  - "https://www.codecademy.com/learn/learn-html"
+---
+
+# HTML - Tables
+
+In HTML, a table is an element that allows for the representation of data in two dimensions; these dimensions are columns and rows.
+
+## Structure
+
+Tables are composed of multiple smaller types of elements that have special meaning within the structure of the table.  The elements that are used in the construction of a typical table are as follows:
+
+### Table
+
+The `<table>` element itself serves as the container for all elements and information contained within the table including headers, columns, and rows.
+
+### Table Row
+
+The table row element, denoted as `<tr>`, is used to display data within a row (horizontally) in a table. Table rows do not display data on their own. Instead, table rows serve as a container for child elements that are responsible for displaying the data in a semantic way.
+
+```html
+<table>
```

Comment:
here instead of adding comments, maybe we can just do:

```html
<table>
   <tr>Row 1</tr>
   <tr>Row 2</tr>
   <tr>Row 3</tr>
  </table>
  ```